x-pack/plugins/security_solution/public/flyout/entity_details/user/header.tsx

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  EuiSpacer,
+  EuiIcon,
+  EuiBadge,
+  EuiText,
+  EuiFlexItem,
+  EuiFlexGroup,
+  useEuiTheme,
+  EuiTitle,
+} from '@elastic/eui';
+
+import React, { useMemo } from 'react';
+import { css } from '@emotion/react';
+import { max } from 'lodash/fp';
+import { SecurityPageName } from '@kbn/security-solution-navigation';
+import { getUsersDetailsUrl } from '../../../common/components/link_to/redirect_to_users';
+import type {
+  ManagedUserData,
+  ObservedUserData,
+} from '../../../timelines/components/side_panel/new_user_detail/types';
+import * as i18n from '../components/translations';
+
+import { SecuritySolutionLinkAnchor } from '../../../common/components/links';
+import { PreferenceFormattedDate } from '../../../common/components/formatted_date';
+import { FlyoutHeader } from '../../shared/components/flyout_header';
+
+interface UserPanelHeaderProps {
+  userName: string;
+  observedUser: ObservedUserData;
+  managedUser: ManagedUserData;
+}
+
+export const UserPanelHeader = ({ userName, observedUser, managedUser }: UserPanelHeaderProps) => {
+  const { euiTheme } = useEuiTheme();
+
+  const lastSeenDate = useMemo(
+    () =>
+      max([observedUser.lastSeen, managedUser.lastSeen].map((el) => el.date && new Date(el.date))),
+    [managedUser.lastSeen, observedUser.lastSeen]
+  );
+
+  return (
+    <FlyoutHeader data-test-subj="user-panel-header">
+      <EuiFlexGroup gutterSize="s" responsive={false} direction="column">
+        <EuiFlexItem grow={false}>
+          <EuiText size="xs" data-test-subj={'user-panel-header-lastSeen'}>
+            {lastSeenDate && <PreferenceFormattedDate value={lastSeenDate} />}
+            <EuiSpacer size="xs" />
+          </EuiText>
+        </EuiFlexItem>
+        <EuiFlexItem grow={false}>
+          <EuiFlexGroup gutterSize="s" alignItems="center" responsive={false}>
+            <EuiFlexItem grow={false}>
```

Comment:
Hi @machadoum! I just want to bring your attention to this [PR](https://github.com/elastic/kibana/pull/171261) that extracted the title component to `FlyoutTitle`. It is currently under review though, so it might get in after your PR :)

Reply:
Thank you!

Reply:
I updated the code [here](https://github.com/elastic/kibana/pull/169514/commits/674d521fb11542d9dd2632bf8a6a90f51c1dde44)  to use `FlyoutTitle`.